spec/models/basket_spec.rb

```diff
@@ -3,14 +3,16 @@
 RSpec.describe Basket, type: :model do
   let(:filepath) { Rails.root.join('baskets', 'basket_2.txt') }
   let(:file) { Rack::Test::UploadedFile.new(filepath, 'txt') }
-  let(:basket) { create(:basket) }
+  let(:user) { create(:user) }
+  let(:basket) { create(:basket, user: user) }
```

Comment:
Tip: you can define the association in the factory definition https://github.com/thoughtbot/factory_bot/blob/master/GETTING_STARTED.md#associations